README.md

```diff
@@ -146,124 +117,206 @@ Visibility is an ENUM and can be referenced by string name ("public" | "private"
+
+## `GET /friends`
+
+### Fetch Format
+
+None
+
+### Response
 
-### The Friend Object
+On success, returns a JSON object containing an array of **User Objects**.
 
 ```json
-{
-  "id": 123,
-  "name": "Up to 255 chars",
-  "status": "requested"
-}
+[
+  { "user object": "see the users route" },
+  { "user object": "see the users route" },
+  { "user object": "see the users route" }
+]
 ```
 
-A friend object is just a user object with an extra property "status" that tells the relationship of the friend to the user.\
-status is an ENUM and can be referenced by string name ("requested" | "waiting" | "friends"). The API will always return the string version.
+## `GET /friends/requested`
 
-### Routes
+### Fetch Format
 
-|   Type | Route                | Description                                                                                            |
-| -----: | -------------------- | ------------------------------------------------------------------------------------------------------ |
-|  `GET` | `/friends`           | Returns the current user's friends and people who have requested friendship with the current user      |
-|  `GET` | `/friends/requested` | Returns friends the current user has requested friendship with, but they haven't accepted              |
-|  `GET` | `/friends/:id`       | A temp route that serves real data by specifying a certain user in the url (gets said user's friends). |
-| `POST` | `/friends/:id`       | Set the friendship status from the current user to the target user. See usage below                    |
+None
 
-When user A requests friendship with B, A will see the status "requested". B will see the status "waiting". Once B accepts, both A and B will see the status "friends".\
-To change the status of a friendship using the POST request, one must pass in the status and the target user as a json object.\
+### Response
+
+On success, returns status `200` and JSON array of **User Objects**. Format will be the same as the `GET /friends` route.
+
+## `POST /friends/:id`
+
+### Fetch Format
+
+The body of the request must contain a JSON object with the id of the target user a selected action. Action can be one of ("request" | "accept" | "reject").\
 To request friendship with user 123,
 
 ```json
 {
   "id": "123",
-  "status": "requested"
+  "action": "request"
 }
 ```
 
 To accept friendship with user 9001,
 
 ```json
 {
-  "id": 9001,
-  "status": "accepted"
+  "id": "9001",
+  "action": "accept"
 }
 ```
 
 To reject friendship with user 9001,
 
 ```json
 {
-  "id": 9001,
-  "status": "rejected"
+  "id": "9001",
+  "action": "reject"
 }
 ```
 
-## ---------- Loans ----------
+### Response
+
+On success, return status `200`. On failure, return status `404`.
 
-### The Loans Object
+# Loans
+
+### Loan Object
+
+The timestamps are return as strings\
+Status can be one of ("pending" | "accepted" | "loaned" | "returned" | "late")
```

Comment:
Should remove `late` status (since it's a calculated property) add `canceled` and `declined`. Not sure if we'll end up using those states yet (I'll probably just make declining and canceling use the delete route for now) but in the future we should support them for easier undos.